verilog/tools/ls/vscode/package.json

```diff
@@ -65,9 +72,9 @@
         "mocha": "^9.1.1",
         "ts-loader": "^9.2.5",
         "typescript": "^4.4.3",
+        "vsce": "^1.100.1",
         "webpack": "^5.52.1",
-        "webpack-cli": "^4.8.0",
-        "vsce": "^1.100.1"
+        "webpack-cli": "^4.10.0"
```

Comment:
Is the version bump intentional?

Reply:
not directly, it was the consequence of running it with current versions (that then also modified this package.json file. So I am not entirely sure if this file is even needed if it is anyway, which is essentially the question in #1139)